go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/kubernetes-csi/external-attacher
 
-go 1.22.5
+go 1.22.7
```

Comment:
For the record, this is not version of go compiler used to build binaries from this module. It is version of go versioning rules. I can see Xing already pointed you to the right place.

/close